client/pages/game.jsx

```diff
@@ -11,20 +12,40 @@ export default class Game extends React.Component {
     this.state = {
       board: new Board(),
       gamestate: new GameState(),
-      meta: null
+      meta: null,
+      side: null,
+      socket: io()
```

Comment:
You should connect the socket in `componentDidMount` instead of `constructor`. Also, we only store data in state, so you should do `this.socket` instead, like `this.timerId`